### PR TITLE
feat: Add CLI

### PR DESCRIPTION
**Pull request by simenandre**

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { jsonToRuntypes } from '.';
+
+const main = async () => {
+  const prog = program
+    .name('json-to-runtypes')
+    .description(
+      'Convert JSON into Runtypes capable of parsing the same JSON shape.',
+    )
+    .option('-i, --input <input file>', 'Path to JSON. Can also be stdin.')
+    .option('-o, --output <output file>', 'Output file.')
```
Comment (runeh) [inline src/cli.ts]: This is optional, right? It'll do stdout if omitted?

File: README.md [hunk 2]
```diff
@@ -85,3 +85,24 @@ export const RootRt = rt.Record({
...
+Options:
+  -i, --input <input file> Path to JSON. Can also be stdin.
+  -o, --output <output file> Output file.
+  h, --help display help for command
+```
+
+The command line supports pipes:
+
+```shell
+▶ cat my-json-file.json|npx json-to-runtypes -o my-types.ts
```
Comment (runeh) [inline README.md]:
```suggestion
▶ cat my-json-file.json | npx json-to-runtypes -o my-types.ts
```

File: src/cli.ts (hunk 1 above)
Comment (simenandre) [reply src/cli.ts]: Both of them are optional, yes :)

Comment (runeh) [reply src/cli.ts]:
```suggestion
    .option('-o, --output <output file>', 'Output file. stdout if omitted,')
```

**Comment by runeh** [comment]
Woo!

**Pull request merged**